src/Control/Effect/Labelled.hs

```diff
@@ -46,7 +47,17 @@ import Data.Kind (Type)
 --
 -- @since 1.0.2.0
 newtype Labelled (label :: k) (sub :: (Type -> Type) -> (Type -> Type)) m a = Labelled (sub m a)
-  deriving (Alternative, Applicative, Functor, Monad, Fail.MonadFail, MonadIO, MonadPlus, MonadTrans)
+  deriving
+    ( Alternative
+    , Applicative
+    , Functor
+    , Monad
+    , Fail.MonadFail
+    , MonadFix -- ^ @since 1.1.1
+    , MonadIO
+    , MonadPlus
+    , MonadTrans
+    )
```

Comment:
I think breaking this into multiple lines is cleaner than an inline comment `…, MonadFix {- ^ @since 1.1.1 -}, …`